addon/mixins/sortable-item.js

```diff
@@ -308,14 +308,18 @@ export default Mixin.create({
     const groupDirection = this.get('group.direction');
     let dragOrigin;
     let elementOrigin;
+    let scrollOrigin;
 
     if (groupDirection === 'x') {
       dragOrigin = getX(startEvent);
       elementOrigin = this.get('x');
+      scrollOrigin = $(this.element).offset().left;
 
       return event => {
         let dx = getX(event) - dragOrigin;
-        let x = elementOrigin + dx;
+        let elementX = this.get('x') - this.element.offsetLeft;
```

Comment:
Yes, `offsetLeft` returns the distance relative to the offset parent, and will also exclude transforms, while `$(element).offset().left` returns the distance relative to the document.
So by subtracting `this.element.offsetLeft` from `this.get('x')` you basically get the transform offset. I agree that this is kind of a hack and hard to follow - I think I should instead store the original position somewhere every time we set `x`/`y`.

I'm doing this because in order to find the difference in scroll position, the reference point must be the same. If I just do `scrollOrigin - $(this.element).offset().left`, this is not going to yield the scroll offset, because the `left` property will also include the transform previously set to the element.

I'm really not sure if there are better ways to do this - would gladly appreciate any input you have :)